front/pages/w/[wId]/workspace/index.tsx

```diff
@@ -49,6 +50,7 @@ export default function WorkspaceAdmin({
   const [workspaceNameError, setWorkspaceNameError] = useState<string>("");
 
   const [isLoading, setIsLoading] = useState(false);
+  const [showProviderModal, setShowProviderModal] = useState(false);
```

Comment:
`nit`

We usually go with `isXXX` to represent booleans.

```suggestion
  const [isProviderModalOpened, setIsProviderModalOpened] = useState(false);
```

Reply:
Well I'll actually reverting the changes to that file, as we don't want to display the modal for now.